src/components/ActionInput/ActionInput.vue

```diff
@@ -68,6 +68,7 @@ For the multiselect component, all events will be passed through. Please see the
 					:value="value"
 					:placeholder="text"
 					:disabled="disabled"
+					:class="{ focusable: isFocusable }"
```

Comment:
It won't work, it needs to be on the input so that the browser canfocus the proper element :/

Reply:
Did you test it? For me it works (Firefox). I think it works due to https://github.com/shentao/vue-multiselect/blob/d8d29e7714449f4fca4bfe10d3bd21e72c46c9b7/src/Multiselect.vue#L61

Reply:
sorry, I meant due to https://github.com/shentao/vue-multiselect/blob/d8d29e7714449f4fca4bfe10d3bd21e72c46c9b7/src/Multiselect.vue#L5

Reply:
You cannot select anything inside.
It properly opens it, but it's not fully working :)

But I don't think it's an issue with the component itself in the end, but more like a keydown event conflict. Once it's focused, we should prevent anything from opening.
The datepicker should not open on focus. Maybe just let the user enter a date or something. :shrug: 
And the multiselect is a tricky one too. Maybe have the search directly set to true so that keyboard users can directly find what they're looking for?
I don't know :shrug: